test/extension-developer-e2e/setup.sh

```diff
@@ -74,6 +74,15 @@ reg_bundle_img="${LOCAL_REGISTRY_HOST}/bundles/registry-v1/registry-bundle:v0.0.
 catalog_img="${CATALOG_IMG}"
 reg_pkg_name="${REG_PKG_NAME}"
 
+
+########################################
+# Remove the default OperatorHub.io 
+# ClusterCatalog
+########################################
+
+# We only want to worry about the test catalogs
+kubectl delete clustercatalog operatorhubio
+
```

Comment:
I would prefer a way for this not to have to be here. i.e. have a way to parameterize `install.tpl.sh` about whether to actually install the default catalog, rather than always installing it, then removing it...
Would it make more sense to do:

```
kubectl delete -f "https://github.com/operator-framework/catalogd/releases/download/${catalogd_version}/default-catalogs.yaml"
```
?

Reply:
Oh yeah I could do something like pass in another variable to the install.tpl.sh that determines if the install script installs default-catalogs.yaml

Reply:
Do you think it would make more sense to have a variable like NO_DEFAULT_CATALOGS where if that's empty or not set then we install default-catalogs.yaml, or a positive variable like INSTALL_DEFAULT_CATALOGS=true that is set to true by default and then is set to false for the ext-dev-e2e test?

Reply:
Thanks. I'd prefer the later "INSTALL_DEFAULT_CATALOG=true" or "False"... I dislike double-negatives. :)